cristin-intermediate-storage/src/test/resources/cristinUnits/units7.json

```diff
@@ -0,0 +1,5320 @@
+[ {
+  "cristin_unit_id" : "20903.0.0.0",
+  "unit_name" : {
+    "en" : "Omtre AS"
+  },
+  "institution" : {
+    "acronym" : "OMT"
+  },
+  "url" : "https://api.cristin.no/v2/units/20903.0.0.0",
+  "acronym" : "OMT"
+}, {
+  "cristin_unit_id" : "20904.0.0.0",
+  "unit_name" : {
+    "en" : "Prostatakreftforeningen"
+  },
+  "institution" : {
+    "acronym" : "PRO"
+  },
+  "url" : "https://api.cristin.no/v2/units/20904.0.0.0",
+  "acronym" : "PRO"
+}, {
+  "cristin_unit_id" : "20905.0.0.0",
+  "unit_name" : {
+    "en" : "Vista Analyse AS"
+  },
+  "institution" : {
+    "acronym" : "VIS"
+  },
+  "url" : "https://api.cristin.no/v2/units/20905.0.0.0",
+  "acronym" : "VIS"
+}, {
+  "cristin_unit_id" : "20906.0.0.0",
+  "unit_name" : {
+    "en" : "Norwegian Energy Partners"
+  },
+  "institution" : {
+    "acronym" : "NOR"
+  },
+  "url" : "https://api.cristin.no/v2/units/20906.0.0.0",
+  "acronym" : "NOR"
+}, {
+  "cristin_unit_id" : "20907.0.0.0",
+  "unit_name" : {
+    "en" : "REN AS"
```

Comment:
Maybe add all supported languages on query?
https://api.cristin-test.uio.no/v2/doc/index.html#lang

Reply:
I don't need it, but if somebody else need the data its probably ok to have it. I will add it